exception-handling/ExceptionHandling/Program.cs

```diff
@@ -0,0 +1,14 @@
+using ExceptionHandling;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddExceptionHandler<CustomExceptionHandler>();
+
+var app = builder.Build();
+
+app.UseExceptionHandler("/Error");
```

Comment:
what does `/Error` do here? your readme doesn't tell this. document it in readme if it's necessary, otherwise i suggest we remove it from here